pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -218,6 +218,8 @@ private enum ReadOnlyBehavior {
   private final @Nullable String xmlFactoryFactoryClass;
   private @Nullable PGXmlFactoryFactory xmlFactoryFactory;
   private final LazyCleaner.Cleanable<IOException> cleanable;
+  /* this is actually the database we are connected to */
+  private String catalog;
```

Comment:
```suggestion
  private @Nullable String catalog;
```

I guess we have `@SuppressWarning` for `PgConnection` constructor, so it results in silencing warning for the newly added field.
However, the field is nullable, so let's specify nullable type.